pkg/webhook/validate.go

```diff
@@ -293,23 +298,23 @@ func validateNicModel(selector *sriovnetworkv1.SriovNetworkNicSelector, iface *s
 			}
 		}
 		if !sriovnetworkv1.StringInArray(iface.Name, pfNames) {
-			return false
+			return fmt.Errorf("interface name: %s not found in physical function names", iface.PciAddress)
 		}
 	}
 
 	// check the vendor/device ID to make sure only devices in supported list are allowed.
 	if sriovnetworkv1.IsSupportedModel(iface.Vendor, iface.DeviceID) {
-		return true
+		return nil
 	}
```

Comment:
Isn't it also important to log that the vendor/deviceID is not supported?

Reply:
We do log that. This returns true which means that it is supported. When it reaches it reaches the end of the function we log that vendor/device ID is not supported. https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/422/files#diff-8fd5d83d7413d77ee2eab8e5c1a2ea1623d63e2aec69017a95811db9ad9edf52R319